lms/startup.py

```diff
@@ -36,20 +34,12 @@ def run():
     """
     django_db_models_options.patch()
 
-    # We currently use 2 template rendering engines, mako and django_templates,
-    # and one of them (django templates), requires the directories be added
-    # before the django.setup().
-    microsite.enable_microsites_pre_startup(log)
```

Comment:
Is it safe to remove the implementation of this function now?  There are a few tests for it, not sure if you want to adapt those to the new configuration or just remove them.

Reply:
Good point - I'm going to attempt to get rid of it.

Reply:
@jmbowman What's the feeling on tests that change Django settings that are used in other derived settings? Basically, testing the methods used to derive those settings? Is it possible in a test to change a settings and re-import the settings to cause the derived methods to run again?

Reply:
With the current implementation, derivation lambdas go poof once settings are fully loaded, so it's tough to rerun them.  The separately defined derivation functions could still be accessed by name, though; maybe just test those?  Override the settings to change one or more of the settings used in the calculation, then verify that the function does the right thing.

Reply:
Would it help to make use of `importlib.reload` to reload the settings module, forcing all the derivations to re-run? https://docs.python.org/3/library/importlib.html#importlib.reload

Reply:
I've used reload to help reset settings for tests in tubular. It works but then you also need to make sure that the settings are reset again without the changes so the settings wouldn't bleed in to other tests. I would recommend most other options before resorting to using it.